index.html

```diff
@@ -133,7 +133,7 @@ <h2 class="card-title">
                     <li>JavaScript</li>
                     <li>html</li>
                 </ul>
-                <button type="button">
+                <button type="button" class="btn-popup">
                     See Project
                 </button>
             </article>
```

Comment:
- [ ] Please make sure that the popup window is responsive and make sure it looks exactly like the Figma design until 320px you can see from the table below that the background color of the pop window is different also you can see that the margin between the buttons is different than Figma design please fix that

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/74744787/161865611-6ba876d0-c85a-485f-8f1b-5860afc3c53b.png) | ![image](https://user-images.githubusercontent.com/74744787/161865844-dd319b11-fb6a-496b-8478-ae79651cba32.png) |